node/setup.go

```diff
@@ -229,8 +229,8 @@ func createPeerManager(
 		MaxConnectedUpgrade:    maxUpgradeConns,
 		MaxPeers:               maxUpgradeConns + 2*maxConns,
 		MinRetryTime:           250 * time.Millisecond,
-		MaxRetryTime:           5 * time.Minute,
-		MaxRetryTimePersistent: 5 * time.Minute,
+		MaxRetryTime:           2 * time.Minute,
```

Comment:
Reduce retry time to fail fast so that we can move on trying to connect the next peer